regexes/device/mobiles.yml

```diff
@@ -562,6 +562,186 @@ Apple:
+      model: 'MacBook Air 13" (2010)'
+
+    - regex: 'MacBookAir4(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2011)'
+
+    - regex: 'MacBookAir4(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2011)'
+
+    - regex: 'MacBookAir5(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2012)'
+
+    - regex: 'MacBookAir5(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2012)'
+
+    - regex: 'MacBookAir6(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2013-2014)'
+
+    - regex: 'MacBookAir6(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2013-2014)'
+
+    - regex: 'MacBookAir7(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 11" (2015)'
+
+    - regex: 'MacBookAir7(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2015-2017)'
+
+    - regex: 'MacBookAir8(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2018)'
+
+    - regex: 'MacBookAir8(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2019)'
+
+    - regex: 'MacBookAir9(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2020)'
+
+    - regex: 'MacBookAir10(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Air 13" (2020)'
+
+    - regex: 'MacBookPro4(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro (2008)'
+
+    - regex: 'MacBookPro5(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2008)'
+
+    - regex: 'MacBookPro5(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 17" (2009)'
+
+    - regex: 'MacBookPro5(?:%2C|,)[34]'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2009)'
+
+    - regex: 'MacBookPro5(?:%2C|,)5'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2009)'
+
+    - regex: 'MacBookPro6(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 17" (2010)'
+
+    - regex: 'MacBookPro6(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2010)'
+
+    - regex: 'MacBookPro7(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2010)'
+
+    - regex: 'MacBookPro8(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2011)'
+
+    - regex: 'MacBookPro8(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2011)'
+
+    - regex: 'MacBookPro8(?:%2C|,)3'
+      device: 'desktop'
+      model: 'MacBook Pro 17" (2011)'
+
+    - regex: 'MacBookPro9(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2012)'
+
+    - regex: 'MacBookPro9(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2012)'
+
+    - regex: 'MacBookPro10(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 15" (2012-2013)'
+
+    - regex: 'MacBookPro10(?:%2C|,)2'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2012-2013)'
+
+    - regex: 'MacBookPro11(?:%2C|,)1'
+      device: 'desktop'
+      model: 'MacBook Pro 13" (2013-2014)'
```

Comment:
I myself do not know whether to enter devices with two years of release with the same IDs.
I think it's better to leave the device with the very first mention by date.

`model: 'MacBook Pro 13" (2013)'` 

https://support.apple.com/HT201300